praw/models/reddit/submission.py

```diff
@@ -53,6 +53,14 @@ def comments(self):
            submission.comments.replace_more(limit=0)
            comments = submission.comments.list()
 
+        Sort order and comment limit can be set with :attr:`.comment_sort` and
```

Comment:
Looks like sphinx is having an issue with these attrs. Unless you want to dig into getting that to work, feel free to use the following format:

    ``comment_sort``
    ``comment_limit``